it/uk/gov/hmrc/agentsubscriptionfrontend/controllers/SubscriptionControllerISpec.scala

```diff
@@ -290,7 +290,7 @@ class SubscriptionControllerISpec extends BaseControllerISpec with SessionDataMi
         val result = await(controller.submit("addr1")(authenticatedRequest()))
 
         status(result) shouldBe 200
-        checkHtmlResultWithBodyText(result, htmlEscapedMessage("error.postcode.blacklisted"))
+        bodyOf(result) should include("Sorry, the address is not in a format that our system can accept. Please change the address to fix the problems described below:")
```

Comment:
Using `checkHtmlResultWithBodyText` has the advantage that the tests won't need to be updated if the message is changed.